servo-tidy.toml

```diff
@@ -5,6 +5,9 @@ check-ordered-json-keys = [
   "./resources/prefs.json",
   "./resources/package-prefs.json",
 ]
+lint-scripts = [
+  "./python/servo/lints/wpt_lint.py",
```

Comment:
I don't think it's worth changing (up to you), but another option instead of just specifying a Python file would be to use a Python path to the class file. For example:

```toml
lint-scripts = [
    'python.servo.lints.wpt_lint.SomeLint',
]
```